uPortal-api/uPortal-api-rest/build.gradle

```diff
@@ -1,5 +1,7 @@
 description = "Apereo uPortal REST APIs"
 
+apply plugin: 'idea'
```

Comment:
the idea plugin is being applied from https://github.com/Jasig/uPortal/blob/7650ffa9bbc01d28691e7bc550d2137acd6ac242/build.gradle#L80